libs/common-system/src/main/java/org/codice/ddf/configuration/AbsolutePathResolver.java

```diff
@@ -40,13 +60,20 @@ public String getPath() {
     Path absolutePath = Paths.get(path);
 
     if (!absolutePath.isAbsolute()) {
-      if (System.getProperty("ddf.home") != null) {
-        absolutePath = Paths.get(System.getProperty("ddf.home"), path);
+      if (StringUtils.isNotBlank(rootPath)) {
+        absolutePath = Paths.get(rootPath, path);
       } else {
-        LOGGER.warn(
-            "Path {} is relative. System property ddf.home is not set, resolving path to: {} ",
-            path,
-            absolutePath.toAbsolutePath());
+        if (System.getProperty("ddf.home") == null) {
```

Comment:
❗️ No. The log message for the property being null belongs in the overloaded method, not here.